build/yaml/webhook/service.yaml

```diff
@@ -0,0 +1,20 @@
+
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app.kubernetes.io/name: service
+    app.kubernetes.io/instance: webhook-service
+    app.kubernetes.io/component: webhook
+    app.kubernetes.io/created-by: nsx-operator
+    app.kubernetes.io/part-of: nsx-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: subnetset
+  namespace: vmware-system-nsx
+spec:
+  ports:
+    - port: 443
+      protocol: TCP
+      targetPort: 9443
```

Comment:
we cannot just use 9443 port, this port is used in WCP for other purpose.
Suggestion:
Add the below 2 config options in pkg/config
flag.IntVar(&managerOpts.WebhookServerPort, "webhook-server-port", controller.DefaultWebhookServerPort, "Port number to expose the controller webhook server")
	flag.StringVar(&managerOpts.WebhookCertDir, "webhook-cert-dir", controller.DefaultWebhookCertDir, "Directory for certificate for webhook server")

Then in cmd/main.go
add these 2 `WebhookCertDir` `WebhookServerPort`  in
mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
		Scheme:                  scheme,
		HealthProbeBindAddress:  config.ProbeAddr,
		Metrics:                 metricsserver.Options{BindAddress: config.MetricsAddr},
		LeaderElection:          cf.HAEnabled(),
		LeaderElectionNamespace: nsxOperatorNamespace,
		LeaderElectionID:        "nsx-operator",
	})

Reply:
we can use 9981 for the default value of WebhookServerPort.

Reply:
Done, thanks.